### PR TITLE
Overhaul Integrations page, add integration page

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Removes config entries from the integrations page, they are moved to the specific integration page.

CleanShot 2023-05-27 at 15 18 00

CleanShot 2023-05-27 at 15 18 35

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bdraco** [comment]
This is really nice for integrations that have a lot of config entries. ❤️ 

The discoverability of an integration being a custom one is going to help squash some issues before they even get created 👍  The depends on the cloud being more visible is also nice!

It would be great if disabled integrations were at the bottom of `/integrations/integration/{integration}` instead of above enabled ones.

File: src/panels/config/energy/components/ha-energy-grid-settings.ts [hunk 1]
```diff
@@ -229,9 +229,7 @@ export class EnergyGridSettings extends LitElement {
                   })}
                 />
                 <span class="content">${this._co2ConfigEntry.title}</span>
-                <a
-                  href=${`/config/integrations#config_entry=${this._co2ConfigEntry.entry_id}`}
-                >
+                <a href="/config/integrations/integration/co2signal">
```
Comment (piitaya) [inline src/panels/config/energy/components/ha-energy-grid-settings.ts:232]: Use `this._co2ConfigEntry.domain` instead of hardcoded domain

**Review by piitaya — COMMENTED** [review]
The `domain` filter on entity and devices dashboard doesn't work. (empty devices list and not filtered entities list). Is there something missing?

**Comment by bramkragten** [comment]
> The domain filter on entity and devices dashboard doesn't work. (empty devices list and not filtered entities list). Is there something missing?

It is working fine here?

**Comment by Mariusthvdb** [comment]
looking forward to this!
could anything be done to those ugly word-wraps?

**Review by piitaya — DISMISSED** [review]

**Pull request merged**